ntbs-service/Middleware/AuditGetRequestMiddleware.cs

```diff
@@ -43,7 +44,7 @@ public async Task InvokeAsync(HttpContext context, IAuditService auditService)
                     // Fallbacks if user doesn't have an email associated with them - as is the case with our test users
                     if (string.IsNullOrEmpty(userName)) userName = context.User.Identity.Name;
                     // TODO: Differentiate between Cluster and Full view.
-                    await auditService.OnGetAuditAsync(id, model: "Notification", viewType: "Full", userName: userName);
+                    await auditService.OnGetAuditAsync(id, model: "Notification", auditDetails: AuditType.Full, userName: userName);
```

Comment:
again, as this field populates the auditDetails column in the audit db, I thought it's name should be defined in AuditType rather than hardcoded here

Reply:
Nice change!